cpp/src/parquet/metadata.cc

```diff
@@ -1462,40 +1462,45 @@ class ColumnChunkMetaDataBuilder::ColumnChunkMetaDataBuilderImpl {
     column_chunk_->meta_data.__set_total_compressed_size(compressed_size);
 
     std::vector<format::Encoding::type> thrift_encodings;
+    std::vector<format::PageEncodingStats> thrift_encoding_stats;
+    // Force Emplace encoding for RL/DL.
+    thrift_encodings.push_back(ToThrift(Encoding::RLE));
```

Comment:
Is this true? I think `BIT_PACKED` is still supported.

Reply:
https://github.com/apache/arrow/blob/6d2df074e624a6a4462a1539be8dadf19cf39df4/cpp/src/parquet/column_writer.cc#L182-L186

Reply:
Oops, I just keep the logic here as before. It's ok to add or remove it :-)